### PR TITLE
add `id_company` to `ald_demo`

### PR DESCRIPTION
**Pull request by jdhoffa**
This PR adds a new column `id_company` to `ald_demo`, which will include AR's unique IDs for each company. This closes the `r2dii.data` related portion of #197.  

Relates to #197 
Thanks @vintented

### CODE REVIEW

File: tests/testthat/test-all.R [hunk 1]
```diff
@@ -8,3 +8,14 @@ test_that("no data inherits spec_tbl_df", {
...
+test_that("no data is grouped", {
+  library(r2dii.data)
+  is_grouped <- vapply(
+    enlist_datasets("r2dii.data"),
+    function(.x) inherits(.x, "grouped_df"),
+    logical(1)
+  )
+
+  expect_false(any(is_grouped))
+})
```
Comment (jdhoffa) [inline tests/testthat/test-all.R:21]: I also added a test here to ensure that none of the dataframes output by `r2dii.data` are grouped.

**Comment by jdhoffa** [comment]
@vintented if you have time, would appreciate if you could have a quick look at this and make sure it's what you expect.

**Comment by maurolepore** [comment]
Following up at https://github.com/2DegreesInvesting/r2dii.data/pull/216
